service/src/test/java/uk/nhs/adaptors/gp2gp/uat/EhrExtractUATTest.java

```diff
@@ -146,62 +105,65 @@ public void setUp() {
+        inputJsonFileContent = removeEmptyDescriptions(inputJsonFileContent);
+        final Bundle bundle = new FhirParseService().parseResource(inputJsonFileContent, Bundle.class);
+
         messageContext.initialize(bundle);
 
-        EhrExtractTemplateParameters ehrExtractTemplateParameters = ehrExtractMapper.mapBundleToEhrFhirExtractParams(
-            getGpcStructuredTaskDefinition,
-            bundle);
-        String ehrExtractContent = ehrExtractMapper.mapEhrExtractToXml(ehrExtractTemplateParameters);
-        var hl7TranslatedResponse = outputMessageWrapperMapper.map(
-            getGpcStructuredTaskDefinition,
-            ehrExtractContent);
+        final EhrExtractTemplateParameters ehrExtractTemplateParameters =
+            ehrExtractMapper.mapBundleToEhrFhirExtractParams(getGpcStructuredTaskDefinition, bundle);
+
+        final String ehrExtractContent = ehrExtractMapper.mapEhrExtractToXml(ehrExtractTemplateParameters);
+
+        final String hl7TranslatedResponse = outputMessageWrapperMapper.map(getGpcStructuredTaskDefinition, ehrExtractContent);
 
         assertThat(hl7TranslatedResponse).isEqualTo(expectedJsonToXmlContent);
     }
 
     private static Stream<Arguments> testValueFilePaths() {
-        return Stream.of(
-            Arguments.of(INPUT_JSON_WITH_PATHOLOGY_RECORD, OUTPUT_XML_USES_PATHOLOGY_RECORD),
-            Arguments.of(INPUT_JSON_WITH_MEDICATION_RECORD, OUTPUT_XML_USES_MEDICATION_RECORD),
-            Arguments.of(INPUT_JSON_WITH_STRUCTURED_CONSULTATIONS_INCLUDING_NARRATIVE_CONTENT,
-                OUTPUT_XML_USES_STRUCTURED_CONSULTATIONS_INCLUDING_NARRATIVE_CONTENT),
-            Arguments.of(INPUT_JSON_WITH_GENERAL_MISCELLANEOUS_CODED_OBSERVATION_CONTENT,
-                OUTPUT_XML_USES_GENERAL_MISCELLANEOUS_CODED_OBSERVATION_CONTENT),
-            Arguments.of(INPUT_JSON_WITH_BLOOD_PRESSURES, OUTPUT_XML_USES_BLOOD_PRESSURES),
-            Arguments.of(INPUT_JSON_WITH_IMMUNISATIONS, OUTPUT_XML_USES_IMMUNISATIONS),
-            Arguments.of(INPUT_JSON_WITH_DRUG_AND_NON_DRUG_ALLERGIES, OUTPUT_XML_USES_DRUG_AND_NON_DRUG_ALLERGIES),
-            Arguments.of(INPUT_JSON_WITH_DIARY_ENTRIES, OUTPUT_XML_USES_DIARY_ENTRIES),
-            Arguments.of(INPUT_JSON_WITH_REFERRALS, OUTPUT_XML_USES_REFERRALS),
-            Arguments.of(INPUT_JSON_WITH_PROBLEMS_AND_PROBLEM_LINKAGES, OUTPUT_XML_USES_PROBLEMS_AND_PROBLEM_LINKAGES),
-            Arguments.of(INPUT_JSON_WITH_NON_SNOMED_CODED_DATA_VIA_EGTON_CODES, OUTPUT_XML_USES_NON_SNOMED_CODED_DATA_VIA_EGTON_CODES)
+        return Stream.of(Arguments.of("9465701483_Dougill_full_20210119.json", "9465701483_Dougill_full_20210119.xml"),
+            Arguments.of("9465701483_Nel_full_20210119.json", "9465701483_Nel_full_20210119.xml"),
+            Arguments.of("9465701459_Nel_full_20210119.json", "9465701459_Nel_full_20210119.xml"),
+//            Arguments.of("9465698679_Gainsford_full_20210119.json", "9465698679_Gainsford_full_20210119.xml"),
```

Comment:
Commented out due to:
```
java.lang.NullPointerException
	at uk.nhs.adaptors.gp2gp.ehr.utils.DateFormatUtil.toDateTypeTime(DateFormatUtil.java:81)
	at uk.nhs.adaptors.gp2gp.ehr.mapper.EhrFolderEffectiveTime.updateEffectiveTimeLowFormatted(EhrFolderEffectiveTime.java:25)
	at uk.nhs.adaptors.gp2gp.ehr.mapper.NonConsultationResourceMapper.mapResourceToEhrComposition(NonConsultationResourceMapper.java:97)
```
Relates to recently merged https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/166